Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -138,6 +142,15 @@ extension KeyboardShortcuts {
 				focus()
 			}
 		}
+		
+		public func changeWidth(for size: NSControl.ControlSize) {
```

Comment:
This should not be a public method. The control size should be handled like any other control. I guess, you could override `.controlSize` and handle the changes there.